Tests/ComposableArchitectureTests/StoreTests.swift

```diff
@@ -153,38 +189,40 @@ final class StoreTests: BaseTCATestCase {
     let viewStore2 = ViewStore(store2, observe: { $0 })
     let viewStore3 = ViewStore(store3, observe: { $0 })
     let viewStore4 = ViewStore(store4, observe: { $0 })
+    defer {
+      _ = viewStore1
+      _ = viewStore2
+      _ = viewStore3
+      _ = viewStore4
+    }
 
-    XCTAssertEqual(numCalls1, 1)
-    XCTAssertEqual(numCalls2, 1)
-    XCTAssertEqual(numCalls3, 1)
-
-    viewStore4.send(())
-
-    XCTAssertEqual(numCalls1, 2)
-    XCTAssertEqual(numCalls2, 2)
+    XCTAssertEqual(numCalls1, 6)
+    XCTAssertEqual(numCalls2, 4)
```

Comment:
We expect these numbers to creep up, since scopes are composed, but since they closure-based scoping is deprecated, it should almost always be a simple key path access soon.